src/Builder/DatagridBuilder.php

```diff
@@ -140,7 +140,11 @@ public function getBaseDatagrid(AdminInterface $admin, array $values = [])
             $defaultOptions['csrf_protection'] = false;
         }
 
-        $formBuilder = $this->formFactory->createNamedBuilder('filter', 'form', [], $defaultOptions);
+        // NEXT_MAJOR: Remove this line when drop Symfony <2.8 support
+        $formType = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
+            ? 'Symfony\Component\Form\Extension\Core\Type\FormType' : 'form';
```

Comment:
Please use the `::class` as much as you can.

Reply:
Ok! I'll change them all